sass/_theme_rst.sass

```diff
@@ -207,7 +207,12 @@
       line-height: normal
 
     &.literal
-      color: $text-code-color
+      color: inherit
+      background-color: lighten($text-light, 20%)
+      border: 1px solid darken($text-light, 5%)
+      border-radius: $base-line-height / 6
```

Comment:
I'm -1 on making this look like `guilabel`. I don't like rounded borders, and rounded borders aren't used in theme anywhere except the search box and guilabel -- which is a rare role anyways.